darts/models/forecasting/forecasting_model.py

```diff
@@ -285,6 +285,13 @@ def output_chunk_length(self) -> Optional[int]:
         """
         return None
 
+    @property
+    def output_chunk_shift(self) -> Optional[int]:
+        """
+        Number of time steps that the output/prediction starts after the end of the input.
+        """
+        return 0
```

Comment:
the output should not be `None`? To be consistent with `output_chunk_length(self) -> Optional[int]` ?

or the `return` type should be `Optional`.

Reply:
👍